### PR TITLE
Update PayJoin to 2.0

### PR DESCRIPTION
**Pull request by MarnixCroes**

### CODE REVIEW

File: docs/using-wasabi/PayJoin.md [hunk 1]
```diff
@@ -13,14 +13,21 @@
...
-3. Select the coins you want to send.
+3. Click `Continue`.
 
-4. Verify the information is correct, type in the password, and press `Send`.
+4. Enter the label.
+
+5. At `Preview Transaction`, verify that the information is correct.
+The PayJoin indicator is displayed at the top right.
+
+![PayJoin indicator](/PayJoinIndicator.png "PayJoin indicator")
```
Comment (yahiheb) [inline docs/using-wasabi/PayJoin.md:28]: I would replace this image with one that you used for `PayJoinSend.png`

**Review by yahiheb — CHANGES_REQUESTED** [review]
I would replace the `PayJoinSend.png` image with the following one:

asasa

File: docs/using-wasabi/PayJoin.md (hunk 1 above)
Comment (MarnixCroes) [reply docs/using-wasabi/PayJoin.md:28]: [a41c9c6](https://github.com/zkSNACKs/WasabiDoc/pull/1147/commits/a41c9c6279886dbfdafc65cd2c7c7ca0ca695128)

**Comment by MarnixCroes** [comment]
@yahiheb I fixed your comments

**Review by yahiheb — CHANGES_REQUESTED** [review]
> I would replace the `PayJoinSend.png` image with the following one:
> 
> asasa

How about using this picture, because it has the payjoin indicator?

**Review by MaxHillebrand — APPROVED** [review]

File: docs/using-wasabi/PayJoin.md [hunk 2]
```diff
@@ -13,14 +13,23 @@
...
+5. At `Preview Transaction`, verify that the information is correct.
+![PayJoinPreviewTransaction](/PayJoinPreviewTransaction.png "PayJoin Preview Transaction")
+
+The PayJoin indicator at the top right indicates that this transaction is a PayJoin.
+
+![PayJoin indicator](/PayJoinIndicator.png "PayJoin indicator")
+
+6. Click `Confirm`, and type in the password.
 
 ![Send Bitcoin PayJoin transaction in Wasabi Wallet](/PayJoinSend.png "Send Bitcoin PayJoin transaction in Wasabi Wallet")
```
Comment (yahiheb) [inline docs/using-wasabi/PayJoin.md:34]: This should be below the 2nd or 3rd point IMO

**Review by yahiheb — CHANGES_REQUESTED** [review]

File: docs/using-wasabi/PayJoin.md [hunk 3]
```diff
@@ -13,14 +13,23 @@
...
 
-4. Verify the information is correct, type in the password, and press `Send`.
+4. Enter the label.
+
+5. At `Preview Transaction`, verify that the information is correct.
+![PayJoinPreviewTransaction](/PayJoinPreviewTransaction.png "PayJoin Preview Transaction")
+
+The PayJoin indicator at the top right indicates that this transaction is a PayJoin.
+
+![PayJoin indicator](/PayJoinIndicator.png "PayJoin indicator")
```
Comment (yahiheb) [inline docs/using-wasabi/PayJoin.md:30]: IMO we don't need this or its image.

**Review by yahiheb — CHANGES_REQUESTED** [review]

File: docs/using-wasabi/PayJoin.md (hunk 2 above)
Comment (MarnixCroes) [reply docs/using-wasabi/PayJoin.md]: that was forgotten because of the renemaing
fixed in latest commit

File: docs/using-wasabi/PayJoin.md (hunk 3 above)
Comment (MarnixCroes) [reply docs/using-wasabi/PayJoin.md]: fixed

**Review by yahiheb — APPROVED** [review]

**Pull request merged**